library/Freckle/Test/Yesod/GeneralAssertions.hs

```diff
@@ -0,0 +1,40 @@
+module Freckle.Test.Yesod.GeneralAssertions
```

Comment:
You'll want to make sure every assertion here has `HasCallStack` all the way down (to include `liftYesodExample`, I believe) . Otherwise, the test failures will be reported as failing at the site of the underlying call(s) in this module, instead of the location of the test.